catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/model/Segment.js

```diff
@@ -26,13 +26,47 @@ define([
 ], function (Backbone, _, FieldDescriptors, Field) {
 
     var counter = 0;
-    var typeCounters= {
+    var typeCounters = {
         Service: 1,
         Organization: 1,
         Content: 1,
         ServiceBinding: 1
     };
 
+    function checkDateCompatibility() {
+        if (!Date.prototype.toISOString) {
+            (function () {
+
+                function pad(number) {
+                    if (number < 10) {
+                        return '0' + number;
+                    }
+                    return number;
+                }
+
+                Date.prototype.toISOString = function () {
```

Comment:
Why do you even need to do this? The only browser that wouldn't have this would be IE8 and below, which we don't even support. IE10+ only